File: C++/123_Number_Flip/main.cpp
//https://binarysearch.com/problems/123-Number-Flip
int solve(int n) {
    string s = to_string(n);
    for(char &c: s){
        if(c != '3'){
            c = '3';
            break;
        }
    }
    return stoi(s);
}

File: Java/123_Number_Flip/main.java
//https://binarysearch.com/problems/123-Number-Flip
import java.util.*;

class Solution {
    public int solve(int n) {
        StringBuilder s = new StringBuilder(String.valueOf(n));
        for(int i = 0; i < s.length(); i++){
            if(s.charAt(i) != '3'){
                s.setCharAt(i, '3');
                break;
            }
        }
        String ans = s.toString();
        return Integer.parseInt(ans);
    }
}